### ISSUE TITLE
TypeAlias is incorrectly reported as not accessed

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1013]
```python
from typing import TypeAlias

A: TypeAlias = "A"
```

**Comment by erictraut** [comment]
This is now implemented in Pyright 1.1.69, which I just published. It will appear in the next Pylance release as well.
